### ISSUE TITLE
how to do gradient accumulation  in bugua

### ISSUE THREAD

**Issue by CaRRotOne** [BaguaSys/bagua#571]
I could use no_sync() with DDP in pytorch to do gradient accumulation. I haven't found related inferface in bagua.

**Comment by wangraying** [comment]
@shjwudp there is an issue related to gradient accumulation.
